loadtest/tests.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/mattermost/mattermost-server/utils"
 )
 
+const (
+	OPENGRAPH_TEST_URL = "https://s3.amazonaws.com/mattermost-load-test-media/index.html"
```

Comment:
Does a fixed URL result in any less stress on the server than might be expected from normally different URIs? (i.e caching?) Could append a random query string if an issue.

Reply:
AFAIK the go HTTP client shouldn't be doing any caching and there's no cloudfront cache in front of it (though s3 might do some sort of caching) so I don't think it should be an issue but we can always revisit if it is